internal/test/framework/windowsvm.go

```diff
@@ -49,9 +49,10 @@ type WindowsVM interface {
 	// CopyFile copies the given file to the remote directory in the Windows VM. The remote directory is created if it
 	// does not exist
 	CopyFile(string, string) error
-	// RetrieveFiles retrieves the list of file from the directory in the remote Windows VM to the local host. As of
-	// now, we're limiting every file in the remote directory to be written to single directory on the local host
-	RetrieveFiles(string, string) error
```

Comment:
Do we really want to remove this method fully and replace it with a recursive version? I'd think we'd want to be able to copy the contents of a directory without recursing.

Reply:
the original method retrieveFiles only copies files, while it skipped all the directories whereas this can handle both the scenarios.